### ISSUE TITLE
file already exists but should not.... cant disable/fix

### ISSUE THREAD

**Issue by Dariusz1989** [pyinstaller/pyinstaller#5043]
Hey

I'm royally stuck with this issue. Here is the relevant part of the code I tried>

'''
for projectPath in jobRange:
    projectPath = projectPath + "**/"
    print("Project path : ", projectPath)
    for f in glob.glob(projectPath + "*.pyd", recursive=True):
        a = QFileInfo(f)
        relative = a.canonicalPath()
        for patToRemove in removePath:
            relative = relative.replace( patToRemove,"")
        bins.append((relative+"/"+a.fileName(),f,""))

print("All Hidden imports : ",hidImports)
for a in bins:
    print (a)

a = Analysis(
    ["starter.py"],
    hookspath=["."],
    cipher=cipher_obj,
    win_private_assemblies=True,
    win_no_prefer_redirects=True,
)
a.datas += bins

for d in a.datas:
    print("Datas : ",d)
    if 'pyconfig' in d[0]:
        a.datas.remove(d) # it finds it and it does nothing...
        print("removed",a.datas)
        #break

print("Clean done")

pyz = PYZ(a.pure,
          a.zipped_data,
          cipher=cipher_obj
          )

options = [('W ignore', None, 'OPTION')] # does nothing

exe = EXE(pyz,
          a.scripts,
          a.binaries,
          a.zipfiles,
          a.datas,
          options, # does nothing 
          name='testApp',
          debug=False,
          uac_admin=True,
          uac_uiaccess=True,
          bootloader_ignore_signals=True,
          strip=False,
          upx=False,
          console=False ,
          clean=True,
          onefile=True,
          )

'''
No matter what I try, I always get that output. How can I remove it? Or where is it located in source code that I could just... "erase that warning from existence"...

TIA!

**Comment by Legorooj** [comment]
What are you trying to do exactly?

**Comment by Dariusz1989** [comment]
When I start the app in noconsole mode I get a pop up window saying that file already exists... I press OK and the app starts. So I want to remove that warning. So far I'm unable to do so...

**Comment by Legorooj** [comment]
@Dariusz1989 please post a screenshot of this popup window.

**Comment by Dariusz1989** [comment]
Pic >
![image](https://user-images.githubusercontent.com/13185728/89099278-f2a39600-d3e5-11ea-8ce2-07f0c8768cd0.png)

**Comment by bwoodsend** [comment]
@Dariusz1989 Does the file already exist?

**Comment by Legorooj** [comment]
Not sure this is a PyInstaller issue. What are your imports @Dariusz1989?

**Comment by Dariusz1989** [comment]
@bwoodsend yes and no... I don't create the file. Pyinstaller does make it when I open the file each time I start the app. Curious tho that its always the same tmp file tho...

@Not sure this is a PyInstaller issue. What are your imports @Dariusz1989?
Imports? Hmmm what do you mean? I import bunch of files I made for the app & pyside2 & other libraries. I have second app with pyside2 test but it has no errors like this. But its 1 file script. This one is about 60 pyd files. 

As to is it pyinstaller issue

https://github.com/pyinstaller/pyinstaller/issues/4561
https://github.com/pyinstaller/pyinstaller/issues/4695
https://stackoverflow.com/questions/19055089/pyinstaller-onefile-warning-pyconfig-h-when-importing-scipy-or-scipy-signal

I went over 50 topics or so. Each giving more fix than other, coming all the way down to editing files inside site-packages pyinstaller location... I'm out of ideas how to disable this warning. Any help would be great.

**Comment by Dariusz1989** [comment]
@bwoodsend yes and no... I don't create the file. Pyinstaller does make it when I open the file each time I start the app. Curious tho that its always the same tmp file tho...

@Legorooj
Imports? Hmmm what do you mean? I import bunch of files I made for the app & pyside2 & other libraries. I have second app with pyside2 test but it has no errors like this. But its 1 file script. This one is about 60 pyd files. 

As to is it pyinstaller issue

https://github.com/pyinstaller/pyinstaller/issues/4561
https://github.com/pyinstaller/pyinstaller/issues/4695
https://stackoverflow.com/questions/19055089/pyinstaller-onefile-warning-pyconfig-h-when-importing-scipy-or-scipy-signal

I went over 50 topics or so. Each giving more fix than other, coming all the way down to editing files inside site-packages pyinstaller location... I'm out of ideas how to disable this warning. Any help would be great.

**Comment by Legorooj** [comment]
@Dariusz1989 I mean which modules do you import with `import ...` and `from ... import ...`.

**Comment by bwoodsend** [comment]
@Dariusz1989 In future please don't duplicate issues onto the support forum. We have enough clutter as it is without needless duplicity.

**Comment by Dariusz1989** [comment]
@Legorooj sys, os, most of pyside2 and about 60 ones I made in the bigger app... do you want me to list all of them? In imports as far as I can print the "generic ones" are > Analysis.datas skipping the custom ones.


```
Datas :  ('base_library.zip', 'S:\\_2Out_Compiled\\installer\\build\\pyinstaller\\base_library.zip', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_zh_TW.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_zh_TW.qm', 'DATA')
Datas :  ('lib2to3\\Grammar.txt', 'C:\\Program Files\\Python37\\lib\\lib2to3\\Grammar.txt', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_fi.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_fi.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_cs.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_cs.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_es.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_es.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_uk.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_uk.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_he.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_he.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_pl.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_pl.qm', 'DATA')
Datas :  ('lib2to3\\PatternGrammar.txt', 'C:\\Program Files\\Python37\\lib\\lib2to3\\PatternGrammar.txt', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_ja.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_ja.qm', 'DATA')
Datas :  ('lib2to3\\Grammar3.7.4.final.0.pickle', 'C:\\Program Files\\Python37\\lib\\lib2to3\\Grammar3.7.4.final.0.pickle', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_en.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_en.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_it.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_it.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_sk.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_sk.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_ru.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_ru.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_de.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_de.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_fr.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_fr.qm', 'DATA')
Datas :  ('PySide2\\qt.conf', 'S:\\pyenv\\lib\\site-packages\\PySide2\\qt.conf', 'DATA')
Datas :  ('lib2to3\\tests\\data\\README', 'C:\\Program Files\\Python37\\lib\\lib2to3\\tests\\data\\README', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_ar.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_ar.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_lv.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_lv.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_bg.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_bg.qm', 'DATA')
Datas :  ('lib2to3\\PatternGrammar3.7.4.final.0.pickle', 'C:\\Program Files\\Python37\\lib\\lib2to3\\PatternGrammar3.7.4.final.0.pickle', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_hu.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_hu.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_da.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_da.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_ko.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_ko.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_gd.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_gd.qm', 'DATA')
Datas :  ('PySide2\\translations\\qtbase_ca.qm', 'S:\\pyenv\\lib\\site-packages\\PySide2\\translations\\qtbase_ca.qm', 'DATA')
Datas :  ('Include\\pyconfig.h', 'C:\\Program Files\\Python37\\Include\\pyconfig.h', 'DATA')

```


I've now done a fresh windows format, and the app no longer starts after I build it, I just get `pyinstaller failed to execute script pyiboot01_bootstrap` so that's another issue I'm trying to debug now... I still get the 1st error tho.

@bwoodsend I don't know if its the same issue. With latest tries I'm getting new errors now :- (

**Comment by Legorooj** [comment]
@Dariusz1989 can we have that same output but for `Analysis.binaries`?

**Comment by Dariusz1989** [comment]
Hey

I think I managed to solve it... "For now" Since I'm using pycharm... and I have multiple projects loaded in 1 "project" as I attached extra project to main one, pycharm runs some sort of "add environments" so they all see each other with easy pathing... and importing. But apaprently that mess up with PyInstaller... 

I ended up adding sys.paths.append() envi of root of all the projects and then pyinstaller run with no issues. I also disconnected pyinstaller project inside pycharm from other ones that I was building. Thanks to the envi var I was able to import the main module from other project and everything seem to just "run" 

The only issue I have now is with `pyinstaller failed to execute script pyiboot01_bootstrap` which is due to me trying to use Cryptodomex and not Crypto... I know Pyinstaller had it as hook, but I have no idea how to configure it properly... any ideas? Should I make new topic ?

My current config.spec has this >
```
if SECRET_CIPHER:
    cipher_obj = PyiBlockCipher(key=SECRET_CIPHER)
    print("Encrypting! ",cipher_obj)
else:
    cipher_obj = None
```
Inside `from PyInstaller.archive.pyz_crypto import PyiBlockCipher` I have renamed `'Crypto.Cipher._AES'` to `'Cryptodome.Cipher._AES'`

Not sure how to bite it :c I can't use Crypto, or at least I don't want to install msvc2017 on fresh system build just to get crypto :/

**Comment by Legorooj** [comment]
@Dariusz1989 use the development version and `tinyaes` if you want up to date crypto support.
